projects/components/src/summary-box/summary-box.component.ts

```diff
@@ -0,0 +1,55 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
+import { Color } from '@hypertrace/common';
+
+@Component({
+  selector: 'ht-summary-box',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `<div class="ht-summary-box">
+    <span
+      class="summary-text"
+      *ngIf="this.count > 0"
+      [htTooltip]="summaryTooltip"
+      [style.color]="this.color"
+      [style.backgroundColor]="this.backgroundColor"
+      [ngClass]="this.displayStyle"
+      >{{ this.summaryText }} {{ this.suffix }}</span
+    >
+    <ng-template #summaryTooltip>
```

Comment:
take string | number | TemplateRef as input and supply that directly to `htTooltip`. This component shouldn't be constructing its own tooltip.

Reply:
done